arc_design_contest/2019/HUST_SoundSnake/src/makefile

```diff
@@ -0,0 +1,26 @@
+# Application name
+APPL ?= SoundSnake
+
+#APPL_DEFINES = -DUSE_APPL_MEM_CONFIG -DV2DSP_XY -DMODEL_BIT_DEPTH=8
+
+#
+# root dir of embARC
+#
+EMBARC_ROOT = ../../..
```

Comment:
project folder should be embarc_osp\embarc_applications\arc_design_contest\2019\HUST_SoundSnake\src

so the EMBARC_ROOT should be ../../../../..